cnm/ipam/ipam.go

```diff
@@ -267,3 +268,35 @@ func (plugin *ipamPlugin) releaseAddress(w http.ResponseWriter, r *http.Request)
 
 	log.Response(plugin.Name, &resp, err)
 }
+
+// Handles retrieving info about ipaddress statistics
+func (plugin *ipamPlugin) getPoolInfo(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can you please move this function to right after releasePoolPath, so that pool APIs stay together?